tests/integration/backup_tests/test_backups.py

```diff
@@ -362,15 +356,18 @@ async def test_restore(ops_test: OpsTest, continuous_writes_to_db) -> None:
 
```

Comment:
A few lines above (line 352) - There is a reference to a non-existing function: `ha_helpers.count_writes(ops_test)`